classList.js

```diff
@@ -106,7 +106,7 @@ classListProto.add = function () {
 	;
 	do {
 		token = tokens[i] + "";
-		if (~checkTokenAndGetIndex(this, token)) {
+		if (checkTokenAndGetIndex(this, token) === -1) {
```

Comment:
This would be `!== -1`

Reply:
@eligrey No, if is == -1, does not exist, therefore we need to add it.
The test will fail if I change this

Reply:
I believe @diervo is correct.  I just experienced this issue in a project where we're using this fine poly.
Yes, (~x)  == (x !== -1).

But I think some errors were introduced in a [previous commit](https://github.com/eligrey/classList.js/commit/1e80f5082a2b2c497a3b3f6ff692c1e98cfdbe1b), which is what @divero is pointing out.

Line 97 was originally changed  from x !== -1 to !~x.  Which I think should have been ~x.
Line 109 was changed from x === -1 to ~x.  Which I think should have been !~x.  
Line 132 changed from x !== -1 to ~x.  Which I think was proper.

Regardless of whether (in)equality or the complement operator are used, there is a logic error that should be corrected.

Reply:
Addressing this now